### PR TITLE
Make code snippet colorful

### PR DESCRIPTION
**Pull request by Youssef1313**
Specifying the code snippet language (vb) to make reserved words blue and comments green

( Compare the two snippets in the picture )
image

### CODE REVIEW

**Review by rpetrusha — APPROVED** [review]
This looks good, @Youssef1313. Thank you for making the additional changes to improve the formatting of the entire article. I'll merge your PR as soon as our internal build completes successfully.

File: docs/visual-basic/programming-guide/language-features/objects-and-classes/how-to-infer-property-names-and-types-in-anonymous-type-declarations.md [hunk 1]
```diff
@@ -37,11 +37,10 @@ Anonymous types provide no mechanism for directly specifying the data types of p
      The resulting type for `anon` would have one property, `Book`, of type <xref:System.Collections.IEnumerable>(Of XElement).  
   
 - From a function that has no parameters, such as `SomeFunction` in the following example.  
-  
-     `Dim sc As New SomeClass`  
-  
-     `Dim anon1 = New With {Key sc.SomeFunction()}`  
-  
+  ```vb
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/objects-and-classes/how-to-infer-property-names-and-types-in-anonymous-type-declarations.md]:
```suggestion

  ```vb
```

File: docs/visual-basic/programming-guide/language-features/objects-and-classes/how-to-infer-property-names-and-types-in-anonymous-type-declarations.md [hunk 2]
```diff
@@ -51,20 +50,18 @@ Anonymous types provide no mechanism for directly specifying the data types of p
 #### Name inference will fail in many circumstances, including the following:  
   
 - The inference derives from the invocation of a method, a constructor, or a parameterized property that requires arguments. The previous declaration of `anon1` fails if `someFunction` has one or more arguments.  
-  
-     `' Not valid.`  
-  
-     `' Dim anon3 = New With {Key sc.someFunction(someArg)}`  
-  
+    ```vb
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/objects-and-classes/how-to-infer-property-names-and-types-in-anonymous-type-declarations.md]:
```suggestion

    ```vb
```

File: docs/visual-basic/programming-guide/language-features/objects-and-classes/how-to-infer-property-names-and-types-in-anonymous-type-declarations.md [hunk 3]
```diff
@@ -51,20 +50,18 @@ Anonymous types provide no mechanism for directly specifying the data types of p
...
+    ' Not valid.
+    ' Dim anon3 = New With {Key sc.someFunction(someArg)}
+    ```
      Assignment to a new property name solves the problem.  
-  
-     `' Valid.`  
-  
-     `Dim anon4 = New With {Key .FunResult = sc.someFunction(someArg)}`  
-  
+    ```vb
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/objects-and-classes/how-to-infer-property-names-and-types-in-anonymous-type-declarations.md]:
```suggestion

    ```vb
```

File: docs/visual-basic/programming-guide/language-features/objects-and-classes/how-to-infer-property-names-and-types-in-anonymous-type-declarations.md [hunk 4]
```diff
@@ -75,41 +72,37 @@ Anonymous types provide no mechanism for directly specifying the data types of p
      [!code-vb[VbVbalrAnonymousTypes#14](~/samples/snippets/visualbasic/VS_Snippets_VBCSharp/VbVbalrAnonymousTypes/VB/Class1.vb#14)]  
   
 - Inference for multiple properties produces two or more properties that have the same name. Referring back to declarations in earlier examples, you cannot list both `product.Name` and `car1.Name` as properties of the same anonymous type. This is because the inferred identifier for each of these would be `Name`.  
-  
-     `' Not valid.`  
-  
-     `' Dim anon5 = New With {Key product.Name, Key car1.Name}`  
-  
+     ```vb
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/objects-and-classes/how-to-infer-property-names-and-types-in-anonymous-type-declarations.md]:
```suggestion

     ```vb
```

File: docs/visual-basic/programming-guide/language-features/objects-and-classes/how-to-infer-property-names-and-types-in-anonymous-type-declarations.md [hunk 5]
```diff
@@ -75,41 +72,37 @@ Anonymous types provide no mechanism for directly specifying the data types of p
...
      [!code-vb[VbVbalrAnonymousTypes#36](~/samples/snippets/visualbasic/VS_Snippets_VBCSharp/VbVbalrAnonymousTypes/VB/Class1.vb#36)]  
   
      Note that changes in case (changes between uppercase and lowercase letters) do not make two names distinct.  
-  
-     `Dim price = 0`  
-  
-     `' Not valid, because Price and price are the same name.`  
-  
-     `' Dim anon7 = New With {Key product.Price, Key price}`  
+     ```vb
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/objects-and-classes/how-to-infer-property-names-and-types-in-anonymous-type-declarations.md]:
```suggestion

     ```vb
```

File: docs/visual-basic/programming-guide/language-features/objects-and-classes/how-to-infer-property-names-and-types-in-anonymous-type-declarations.md [hunk 6]
```diff
@@ -75,41 +72,37 @@ Anonymous types provide no mechanism for directly specifying the data types of p
...
+     ' Dim anon7 = New With {Key product.Price, Key price}
+     ```
   
 - The initial type and value of one property depends on another property that is not yet established. For example, `.IDName = .LastName` is not valid in an anonymous type declaration unless `.LastName` is already initialized.  
-  
-     `' Not valid.`  
-  
-     `' Dim anon8 = New With {Key .IDName = .LastName, Key .LastName = "Jones"}`  
-  
+     ```vb
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/objects-and-classes/how-to-infer-property-names-and-types-in-anonymous-type-declarations.md]:
```suggestion

     ```vb
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look great @Youssef1313 but there should be an empty line between the text and the code blocks. I left a few suggestions to fix that throughout the text. Thanks!

**Comment by Youssef1313** [comment]
> 
> 
> Changes look great @Youssef1313 but there should be an empty line between the text and the code blocks. I left a few suggestions to fix that throughout the text. Thanks!

@mairaw, Thanks for the suggesstions. I've committed them.

**Review by KathleenDollard — COMMENTED** [review]
Looks good. Thank you!

**Review by mairaw — APPROVED** [review]
Thanks @Youssef1313 for applying the suggested changes. Merging now.

**Pull request merged**